pytube/extract.py

```diff
@@ -263,8 +264,7 @@ def get_ytplayer_config(html: str) -> Any:
     #  and use then load that as json to find PLAYER_CONFIG
     #  inside of it.
     setconfig_patterns = [
-        r"yt\.setConfig\((.*'PLAYER_CONFIG':\s*{.+?})\);",
-        r"yt\.setConfig\((.*\"PLAYER_CONFIG\":\s*{.+?})\);"
+        r"yt\.setConfig\((.*['\"]PLAYER_CONFIG['\"]:\s*{.+?})\);"
```

Comment:
Good catch. At one point while I was experimenting with these regexes I had this, not sure what happened to that.